packages/ipfs/src/core/components/bitswap/stat.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-const Big = require('bignumber.js')
+const Big = require('bignumber.js').default
```

Comment:
To make the typings work, the export signature in [its `.d.ts` file](https://github.com/MikeMcl/bignumber.js/blob/master/bignumber.d.ts)
```ts
export default BigNumber;
export declare class BigNumber ...
```
is either `require('bignumber.js').default` or `require('bignumber.js').BigNumber` in CommonJS

Reply:
I think when namespace and class name do collide in definitions TS does know to not require default, but I could be wrong and it might be behind some config flag. I think I'd still prefer `const { BigNumber: Big } = require('bignumber.js')` over current version, but I'm ok with current version too.